packages/renderer/src/lib/dashboard/DashboardPage.svelte

```diff
@@ -1,18 +1,195 @@
+  const serializableItems = orderedItems.map(item => ({
+    id: item.id,
+    label: item.label,
+    enabled: item.enabled,
+    originalOrder: item.originalOrder,
+  }));
+
+  await tablePersistence.storage.save('dashboard', serializableItems);
+  dashboardPageRegistry.entries = updatedEntries;
+}
+
+// Initialize dashboard on mount
+onMount(async () => {
+  await initializeDashboard();
+});
+
+// Reset function for dashboard layout
+async function resetDashboardLayout(): Promise<void> {
+  if (!tablePersistence.storage) return;
+  try {
+    // Reset using the persistence callbacks (clears saved config)
+    dashboardSections = await tablePersistence.storage.reset('dashboard', defaultSectionNames);
+    dashboardOrdering.clear();
+
+    // Reset the registry to default state
+    setupDashboardPageRegistry();
+
+    // Ensure the registry reflects the reset state
+    if (dashboardPageRegistry.entries.length > 0) {
+      dashboardPageRegistry.entries = convertFromListOrganizerItems(dashboardSections, dashboardPageRegistry.entries);
+    }
+  } catch (error: unknown) {
+    console.error(`Failed to reset dashboard layout: ${error}`);
+  }
+}
+
+// Handle dashboard order changes from ListOrganizer
+function handleDashboardOrderChange(newOrdering: SvelteMap<string, number>): void {
+  dashboardOrdering = new SvelteMap(newOrdering);
+
+  // Save configuration immediately when order changes
+  saveDashboardConfiguration().catch((error: unknown) => {
+    console.error(`Failed to save dashboard configuration after order change: ${error}`);
+  });
+}
+
+// Handle dashboard section toggle changes from LayoutEditor
+function handleDashboardToggle(itemId: string, enabled: boolean): void {
+  dashboardSections = dashboardSections.map(item => (item.id === itemId ? { ...item, enabled } : item));
+
+  // Save configuration immediately when toggle changes
+  saveDashboardConfiguration().catch((error: unknown) => {
+    console.error(`Failed to save dashboard configuration after toggle change: ${error}`);
+  });
+}
 </script>
 
 <NavPage searchEnabled={false} title="Dashboard">
+  {#snippet additionalActions()}
+    <ListOrganizer
+      items={dashboardSections}
+      ordering={dashboardOrdering}
+      title="Configure Dashboard Sections"
+      enableReorder={true}
+      enableToggle={true}
+      onOrderChange={handleDashboardOrderChange}
+      onToggle={handleDashboardToggle}
+      onReset={resetDashboardLayout}
+      resetButtonLabel="Reset Layout"
+    />
+  {/snippet}
+  
   {#snippet content()}
   <div class="flex flex-col min-w-full h-full bg-[var(--pd-content-bg)] py-5">
     <div class="min-w-full flex-1">
       <NotificationsBox />
       <div class="px-5 space-y-5 h-full">
-        {#each dashboardPageRegistry.entries as dashboardRegistryItem (dashboardRegistryItem.id)}
+        {#each sortedDashboardRegistry as dashboardRegistryItem (dashboardRegistryItem.id)}
           <dashboardRegistryItem.component />
         {/each}
```

Comment:
suggestion(blocking): we should use the right synthax, we should use uppercase;


```suggestion
        {#each sortedDashboardRegistry as { id, component: Composant} (dashboardRegistryItem.id)}
          {@const Component = dashboardRegistryItem.component}
          <Component />
        {/each}
```